src/Tasks/GenerateResource.cs

```diff
@@ -2432,6 +2432,12 @@ internal Assembly ResolveAssembly(object sender, ResolveEventArgs args)
                     {
                         if (String.Compare(requestedAssemblyName.Name, candidateAssemblyName.Name, StringComparison.CurrentCultureIgnoreCase) == 0)
                         {
+                            string itemSpec = _assemblyFiles[i].ItemSpec;
+
+                            if (requestedAssemblyName.Name.Contains("System.Runtime"))
```

Comment:
https://gist.github.com/nguerrera/6864d2a907cb07d869be5a2afed8d764

Disclaimer: I wrote this a LONG time ago and it was against a pre-release before the API had taken its final form. I just fixed up the compile errors and did not re-test.

We need this to be an API in the library. I'll submit an API proposal.